turborepo-tests/examples/basic/turbo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "outputs": []
```

Comment:
We should add `../test_example.sh` as an input to all of these pipeline definitions